bedrock/firefox/templates/firefox/whatsnew/index-lite.id.html

```diff
@@ -52,7 +52,7 @@ <h1><img src="{{ static('img/logos/firefox/logo-lite-wordmark-white.png') }}" wi
             message_set='download-firefox-rocket',
             spinner_color='#fff',
             input_label='Masukkan alamat surel dan kirimkan Firefox Lite ke ponselmu.',
-            legal_note_email='Penerima email harus menyetujui sebelumnya.')
+            legal_note_sms='Penerima email harus menyetujui sebelumnya.')
```

Comment:
I believe this should stay as `legal_note_email`, as sms is only supported in the US. There's an outstanding bug on this related to geo-location. See: https://github.com/mozilla/bedrock/issues/6732

TL:DR, I don't think you need to worry about fixing it here, as it's been around for some time. I think we consider this enough of an edge case for it not to be a high priority bug.